handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -1076,21 +1076,31 @@ public void setEnabledProtocols(String[] protocols) {
                 // Enable all and then disable what we not want
                 SSL.setOptions(ssl, SSL.SSL_OP_ALL);
 
+                int opts = SSL.SSL_OP_NO_SSLv2 | SSL.SSL_OP_NO_SSLv3 | SSL.SSL_OP_NO_TLSv1 |
```

Comment:
nit: i guess there is no need to assign this to a temp variable ... consider just passing in the mask directly to `clearOptions`